server/tests/interpreter_test.js

```diff
@@ -606,3 +606,22 @@ exports.testAeca = function(t) {
     runTest(t, 'ASECA: ' + tc[0], src, tc[2]);
   }
 };
+
+exports.testNumber = function(t) {
```

Comment:
It would probably have been sufficient to add a single test into testcases.js, but I won't complain about you writing *more* tests than are needed.